publishing/entry-point-uri/README.md

```diff
@@ -0,0 +1,19 @@
+# Entry point URI
+
+Every blendable web app should have the entry point URI that follows the pattern `/<appname>`, e.g. `/kitchensink` (lowercase).
+
+This URI is automatically linked in Starcounter Administrator.
```

Comment:
It's linked to the start button in the administrator, I think that's what he means